addons/website/models/website_redirect.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class WebsiteRedirect(models.Model):
+    _name = "website.redirect"
+    _description = "Website Redirect"
+    _order = "sequence, id"
+    _rec_name = 'url_from'
+
+    redirect_type = fields.Selection([('301', 'Moved permanently (301)'), ('302', 'Moved temporarily (302)')], string='Redirection Type', required=True, oldname='type')
```

Comment:
```suggestion
    redirect_type = fields.Selection([('301', 'Moved permanently (301)'), ('302', 'Moved temporarily (302)')], string='Redirection Type', required=True, oldname='type', default='301')
```
as for https://github.com/odoo/odoo/commit/b876cb397dbd8fe81a9a0ec0f6efbdc73178b9e7 not ported yet to `master`